src/components/Garden/Feed/ProposalInfo.tsx

```diff
@@ -23,6 +26,9 @@ function ProposalInfo({
 }: ProposalInfoProps) {
   const theme = useTheme()
   const { config } = useGardenState()
+  const { flow, loading: loadingFlow } = useSuperfluidCFAv1(
```

Comment:
wondering what that this `useSuperfluidCFAv1` means and  if we can get a better naming

Reply:
I think that means [Constant Flow Agreement](https://docs.superfluid.finance/superfluid/protocol-overview/in-depth-overview/super-agreements/constant-flow-agreement-cfa) from Superfluid.

Reply:
Yes, @kamikazebr is right. Though I agree with @rperez89 I will change to `useSuperfluid`.